include/exadg/grid/grid.cpp

```diff
@@ -160,10 +160,50 @@ Grid<dim>::create_triangulation(
     {
       // we need to generate the coarse triangulations explicitly
 
-      // TODO
-      // construct all the coarser triangulations
+      // resize the empty coarse triangulations vector
+      coarse_triangulations = std::vector<std::shared_ptr<dealii::Triangulation<dim> const>>(
+        triangulation->n_global_levels());
+
+      unsigned int coarse_level = triangulation->n_global_levels() - 1;
```

Comment:
The name `coarse_level` is misleading, because the lambda `create_coarse_triangulation` refers to the coarsest level. Hence, there is a mismatch in notation. Just use `level` instead, and add a comment that you start, by convention, with the highest level.